ecl/hthor/hthorkey.cpp

```diff
@@ -1263,6 +1267,7 @@ const void * CHThorIndexNormalizeActivity::createNextRow()
     {
         outBuilder.ensureRow();
         size32_t thisSize = helper.transform(outBuilder);
+        callback.finishedRow();
```

Comment:
Is this needed?

Reply:
needs a call to finishedRow in the nextRow loop somewhere, used to have one in the loop at line 1237, now explicit here or if !expanding